source/auth/tests/mongodb-aws.rst

```diff
@@ -2,12 +2,13 @@
 MongoDB AWS
 ===========
 
-There are 5 scenarios drivers MUST test:
+There are 6 scenarios drivers MUST test:
 
 #. ``Regular Credentials``: Auth via an ``ACCESS_KEY_ID`` and ``SECRET_ACCESS_KEY`` pair
 #. ``EC2 Credentials``: Auth from an EC2 instance via temporary credentials assigned to the machine
 #. ``ECS Credentials``: Auth from an ECS instance via temporary credentials assigned to the task
-#. ``Assume Role``: Auth via temporary credentials obtained from an STS AssumeRole request
+#. ``Assume Role``: Auth via temporary credentials obtained from an STS AssumeRoleWithWebIdentity request
+#. ``Assume Role with Web Identity``: Auth via temporary credentials obtained from an STS AssumeRole request
```

Comment:
The above two scenario names do not match their descriptions.

Reply:
Updated